<?xml version="1.0" ?>
<launch>
  <arg name="plan_file" default="src/beb_mr_ctrl/config/traj_point_sim.yaml" />

  <node name="generate_trajectory" type="generate_trajectory.py" pkg="beb_mr_ctrl" output="screen">
    <remap from="setpoint" to="setpoint_pose" />
    <remap from="takeoff" to="ardrone/takeoff" />
    <remap from="land" to="ardrone/land" />
    <remap from="camera_control" to="bebop/camera_control" />
    <rosparam command="load" file="$(arg plan_file)" />
  </node>
</launch>
